### PR TITLE
rb3gen2-corekit, idp confs: add firmware packages

### PR DESCRIPTION
**Pull request by vivpuar**
Adding qcom-qcm6490-{audio, compute} firmware packages
to qcm6490-idp.conf and qcs6490-rb3gen2-core-kit.conf 
to add support of adsp and cdsp firmware

### CODE REVIEW

File: conf/machine/qcs6490-rb3gen2-core-kit.conf [hunk 1]
```diff
@@ -9,3 +9,8 @@ MACHINE_FEATURES = "usbhost usbgadget alsa wifi bluetooth"
 KERNEL_DEVICETREE ?= " \
                       qcom/qcs6490-rb3gen2.dtb \
                       "
+
+MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS += " \
+    linux-firmware-qcom-qcm6490-audio \
+    linux-firmware-qcom-qcm6490-compute \
+"
```
[the remark this answers is not in the document]
Comment (vivpuar) [reply conf/machine/qcs6490-rb3gen2-core-kit.conf:16]: Sure will update these at conf/machine/include/qcom-qcs6490.inc

[the remark this answers is not in the document]
Comment (ndechesne) [reply conf/machine/qcs6490-rb3gen2-core-kit.conf]: I wondered about that too.. even if they are 'core' features, it's still possible for someone to design a platform/machine without audio, or compute.. so i concluded it was ok to leave them in the final machine file instead.

[the remark this answers is not in the document]
Comment (vivpuar) [reply conf/machine/qcs6490-rb3gen2-core-kit.conf:16]: Updated for both in machine files

**Review by quaresmajose — APPROVED** [review]
LGTM

**Review by ndechesne — APPROVED** [review]

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**